api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -43,4 +43,140 @@ object IgnoredParameters {
     case "H2ODeepLearning" => common ++ Seq("pretrained_autoencoder")
     case _ => common
   }
+
+  def ignoredInMOJOs(algorithm: String): Seq[String] = {
+    val common = manuallyImplementedInMOJOModel ++ unimplementedForMOJOs
+    common ++ {
+      algorithm match {
```

Comment:
The most of the fields will have to be ignored directly on algorithms since their existence on particular algorithms doesn't make much sense. This is a consequence of generating SW API from generic V3 classes from H2O-3.